npm/janus.d.ts

```diff
@@ -130,33 +141,65 @@ declare namespace JanusJS {
 			[otherProps: string]: any;
 		};
 		jsep?: JSEP;
-		success?: Function;
-		error?: (error: any) => void;
+		success?: (data?: any) => void;
+		error?: (error: string) => void;
+	}
+	interface Publisher {
```

Comment:
this declaration should not be here, since it is part of the videoroom plugin

Reply:
I agree, this is a bit out of place here. But, where else would you see those plugin-specific stuff? (it's also an option to not have this at all or "later")

Reply:
These declarations should be addressed in separate `d.ts` files, not something to discuss here I guess.